framework/SupervisedLearning/KerasClassifier.py

```diff
@@ -31,16 +31,9 @@
 from scipy import stats
 import os
 
-import utils.lazyImporterUtils
-_tensorflowAvailable = False
-# TensorFlow is optional and Python3 is required in order to use tensorflow for DNNs
-#try:
-# import tensorflow as tf
-tf =  utils.lazyImporterUtils.import_module_lazy_renamed("tf", globals(), "tensorflow")
+import utils.importerUtils
+tf =  utils.importerUtils.importModuleLazyRenamed("tf", globals(), "tensorflow")
 _tensorflowAvailable = True
-  # tf.enable_eager_execution()
-#except ImportError as e:
-#  _tensorflowAvailable = False
```

Comment:
In this modifications, It seems to me we assume TensorFlow is always installed. If this is the case, we should remove the following function. I think an issue may be created to handle this.
```
 def isTensorflowAvailable():
  """
    Return True if tensorflow library is correctly loaded
    @ In, None
    @ Out, _tensorflowAvailable, bool, True if tensorflow library is correctly loaded otherwise False
  """
  return _tensorflowAvailable
```

Reply:
agree

Reply:
removed